grid/recruitment/list-candidates/maps/breezy-hr.suma

```diff
@@ -0,0 +1,234 @@
+profile = "recruitment/list-candidates@1.0"
+provider = "breezy-hr"
+
+map ListCandidates { 
+   call CheckCompanyExists (
+    accessToken = parameters.ACCESS_TOKEN,
+    companyId = parameters.COMPANY_ID,
+  ) { 
+    return map error if (outcome.error) outcome.error
+  }
+
+  call CheckJobExists (
+    accessToken = parameters.ACCESS_TOKEN,
+    companyId = parameters.COMPANY_ID,
+    jobId = input.jobId
+  ) { 
+    return map error if (outcome.error) outcome.error
+  }
+  
+  http GET "/v3/company/{ parameters.COMPANY_ID }/position/{ input.jobId }/candidates" {
+    request {
+      query {
+        page_size = input.limit
+        page = input.page
+      }
+      headers {
+        Authorization = parameters.ACCESS_TOKEN
```

Comment:
Would none of the provider security work for this? (E.g. ApiKeyHeaderSecurity)

Reply:
You are right ApiKeyHeaderSecurity security will work. It should be used here.

Reply:
I will update the provider definition and related maps in separate PR.